Library/Formula/stack.rb

```diff
@@ -0,0 +1,15 @@
+class Stack < Formula
+  desc "A cross-platform program for developing Haskell projects."
+  homepage "https://github.com/commercialhaskell/stack"
+  url "https://github.com/fpco/stack/releases/download/v0.0.0-beta/stack-0.0.0-x86_64-osx.gz"
```

Comment:
Homebrew also distributes binaries; all submitted packages are build using our hardware in a sandbox environment. If you change this formula to build from source, then we'll add the binaries when it's merged.